lib/bundle/vscode_extension_dumper.rb

```diff
@@ -10,7 +10,7 @@ def reset!
 
     def extensions
       @extensions ||= if Bundle.vscode_installed?
-        `code --list-extensions 2>/dev/null`.split("\n").map(&:downcase)
+        `code --list-extensions 2>/dev/null`.split("\n")
```

Comment:
```suggestion
        `code --list-extensions 2>/dev/null`.split("\n").map(&:downcase)
```

makes more sense to me to keep this as-is and avoid having to have callers downcase.

Reply:
Yes, i debated this with myself - and then chose the other implementation - since i felt it's more relevant and explicit when both the arrays are downcased in the same line (causes less of context switching). but, if you feel they can be separated (I agree from a performance perspective), then i will do that change. Just need your thoughts between the 2 reasonings.

Reply:
> but, if you feel they can be separated (I agree from a performance perspective), then i will do that change

Yes please 👍🏻 🙇🏻